### PR TITLE
Package tezos-rust-libs.1.4

### PR DESCRIPTION
**Pull request by dannywillems**
### `tezos-rust-libs.1.4`
Tezos: all rust dependencies and their dependencies

---
* Homepage: https://www.tezos.com/
* Source repo: git+https://gitlab.com/tezos/tezos-rust-libs.git
* Bug tracker: https://gitlab.com/tezos/tezos-rust-libs/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by dannywillems** [comment]
Upstreamed [this commit](add5659f53e2f2e2489a7cc8e6cdee587efae09f) in [tezos-rust-libs](https://gitlab.com/tezos/tezos-rust-libs/-/commit/af962e77732e2ccbde0691f181ae8a4af3d7622e) for future releases. Included to stay consistent with previous tezos-rust-libs releases.

File: packages/tezos-rust-libs/tezos-rust-libs.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Tezos: all rust dependencies and their dependencies"
+maintainer: "contact@tezos.com"
+authors: "Tezos devteam"
+license: "multiple"
```
Comment (kit-ty-kate) [inline packages/tezos-rust-libs/tezos-rust-libs.1.4/opam:5]:
```suggestion
license: "LicenseRef-multiple"
```
is the escape hatch for this kind of thing in the SPDX syntax

Comment (dannywillems) [reply packages/tezos-rust-libs/tezos-rust-libs.1.4/opam]: Thanks. [Upstreaming it](https://gitlab.com/tezos/tezos-rust-libs/-/commit/0b86fd68a92b3aa9281058ed5fb8886d430a8bf9)

**Comment by kit-ty-kate** [comment]
Thanks. Could you return the fix in the opam file upstream?

**Comment by raphael-proust** [comment]
The fix upstream is done: https://gitlab.com/tezos/tezos-rust-libs/-/commit/0b86fd68a92b3aa9281058ed5fb8886d430a8bf9

**Comment by dannywillems** [comment]
Thanks @kit-ty-kate

**Pull request merged**